src/main/utils/utils.ts

```diff
@@ -10,6 +11,30 @@ export class Utils {
     // 1 GB
     private static readonly MAX_SIZE_EXTRACTED_ZIP_BYTES: number = 1000000000;
     private static readonly COMPRESSION_THRESHOLD_RATIO: number = 100;
+    // keep up to 100 logs
+    public static readonly KEEP_LOGS_COUNT: number = 100;
+
+    public static cleanUpOldLogs() {
+        let logFolder: string = ScanUtils.getLogsPath();
+        let logFiles: string[] = fs.readdirSync(logFolder).map(fileName => path.join(logFolder, fileName));
+        let toRemoveCount: number = logFiles.length + 1 - this.KEEP_LOGS_COUNT;
+        let removed: number = 0;
+
+        if (toRemoveCount > 0) {
+            logFiles.sort((lhs, rhs) => fs.statSync(lhs).birthtime.getTime() - fs.statSync(rhs).birthtime.getTime());
```

Comment:
Looks like n*log(n) complexity of I/O operations.
The sorting must be in the memory.